Sources/Plasma/FeatureLib/pfPython/pyGUIControlMultiLineEdit.cpp

```diff
@@ -203,9 +203,12 @@ void pyGUIControlMultiLineEdit::SetEncodedBuffer( PyObject* buffer_object )
         if ( pbmod )
         {
             // something to do here... later
-            uint8_t* daBuffer = nil;
-            Py_ssize_t length;
-            PyObject_AsReadBuffer( buffer_object, (const void**)&daBuffer, &length);
+            Py_buffer view;
+            PyObject_GetBuffer(buffer_object, &view, PyBUF_SIMPLE);
+            uint8_t* daBuffer = (uint8_t*)view.buf;
+            Py_ssize_t length = view.len;
+            PyBuffer_Release(&view);
+
```

Comment:
As part of the scripts update, I *think* I completely removed all uses of buffers because of this deprecation. Maybe we should just remove this binding?